### PR TITLE
Add new maintainers

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
Last week we voted to add new maintainers to the project. Since the vote passed, in this PR I'm updating `MANTAINERS.md` and `CODEOWNERS`. I also invited all 4 new maintainers to have write privileges (request is pending, reason why `CODEOWNERS` validator still report an issue).

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by colega — APPROVED** [review]
🥳

File: MAINTAINERS.md [hunk 1]
```diff
@@ -26,3 +27,7 @@ Some parts of the codebase have other maintainers, the package paths also includ
 - `pkg/ruler`:
   - Josh Abreu Mesa — [@gotjosh](https://github.com/gotjosh) ([Grafana Labs](https://grafana.com/))
   - Steve Simpson - [@stevesg](https://github.com/stevesg) ([Grafana Labs](https://grafana.com/))
+- `packaging`:
+  - Wilfried ROSET - [@wilfriedroset](https://github.com/wilfriedroset) ([OVHCloud](https://www.ovhcloud.com/))
```
Comment (mlcdf) [inline MAINTAINERS.md:31]:
```suggestion
  - Wilfried ROSET - [@wilfriedroset](https://github.com/wilfriedroset) ([OVHcloud](https://www.ovhcloud.com/))
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -26,3 +27,7 @@ Some parts of the codebase have other maintainers, the package paths also includ
...
+  - Nicolas DUPEUX - [@vaxvms](https://github.com/vaxvms) ([OVHCloud](https://www.ovhcloud.com/))
```
Comment (mlcdf) [inline MAINTAINERS.md:32]:
```suggestion
  - Nicolas DUPEUX - [@vaxvms](https://github.com/vaxvms) ([OVHcloud](https://www.ovhcloud.com/))
```

Comment (wilfriedroset) [inline MAINTAINERS.md:32]:
```suggestion
  - Nicolas DUPEUX - [@vaxvms](https://github.com/vaxvms) ([OVHcloud](https://www.ovhcloud.com/))
```

Comment (mlcdf) [reply MAINTAINERS.md:32]:
```suggestion
  - Nicolas DUPEUX - [@vaxvms](https://github.com/vaxvms) ([OVHcloud](https://www.ovhcloud.com/))
```

File: MAINTAINERS.md (hunk 1 above)
Comment (wilfriedroset) [inline MAINTAINERS.md:31]:
```suggestion
  - Wilfried ROSET - [@wilfriedroset](https://github.com/wilfriedroset) ([OVHcloud](https://www.ovhcloud.com/))
```

**Review by mlcdf — CHANGES_REQUESTED** [review]
Hello there, if I may :)

**Review by wilfriedroset — APPROVED** [review]

**Review by vaxvms — APPROVED** [review]

**Review by replay — APPROVED** [review]

**Review by charleskorn — APPROVED** [review]

**Pull request merged**